DESCRIPTION

```diff
@@ -1,13 +1,27 @@
 Package: riycf
 Type: Package
-Title: What the Package Does (Title Case)
+Title: Utility for Calculating Infant and Young Child Feeding (IYCF) Indicators
 Version: 0.1.0
-Author: Who wrote it
-Maintainer: The package maintainer <yourself@somewhere.net>
-Description: More about what it does (maybe more than one line)
-    Use four spaces when indenting paragraphs within the Description.
-License: What license is it under?
+Authors@R: c(
+    person("Nicholus", "Tint Zaw", email = "nicholustintzaw@katilingban.io",
+           role = c("aut", "cre")),
+    person("Ernest", "Guevarra", comment = c(ORCID = "0000-0002-4887-4415"), 
+           email = "ernest@guevarra.io", role = "aut"), 
+    person("Rebecca", "Wolfe", role = "aut"))
+Description: This package aims to generate updated indicators to assess infant 
+    and young child feeding (IYCF) practices at the household level. The 
```

Comment:
why do you say at household level? Isn't these indicators individual level indicators?